File: Week6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

height = -1
while height <= 0 or height > 8:
    height = get_int("Height: ")

for y in range(1, height + 1):
    for x in range(1, height + 1):
        if x > height - y:
            print("#", end="")
        else:
            print(" ", end="")
        if x == height:
            print(" ", "#" * y, end="")
    print("")

File: Week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int y = 0; y < height; y++)
    {
        for (int x = 0; x < height + 3 + y; x++)
        {
            if (x < height && y + x >= height - 1)
            {
                printf("#");
            }

            else if (x > height + 1 && x - y <= height + 2)
            {
                printf("#");
            }

            else
            {
                printf(" ");
            }

        }

        printf("\n");
    }
}